docs/index-en.html

```diff
@@ -0,0 +1,4245 @@
+          </li>
+          <li> <a href="#stereotype" title="https://w3id.org/ontouml#stereotype">stereotype</a>
+          </li>
+          <li> <a href="#subsetsProperty" title="https://w3id.org/ontouml#subsetsProperty">subsetsProperty</a>
+          </li>
+          <li> <a href="#targetEnd" title="https://w3id.org/ontouml#targetEnd">targetEnd</a>
+          </li>
+          <li> <a href="#targetView" title="https://w3id.org/ontouml#targetView">targetView</a>
+          </li>
+          <li> <a href="#topLeftPosition" title="https://w3id.org/ontouml#topLeftPosition">topLeftPosition</a>
+          </li>
+        </ul>
+        <h4>Data Properties</h4>
+        <ul xmlns:extra="https://w3id.org/extra#" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+
+          class="hlist">
+          <li> <a href="#cardinalityValue" title="https://w3id.org/ontouml#cardinalityValue">cardinalityValue</a>
+          </li>
+          <li> <a href="#description" title="https://w3id.org/ontouml#description">description</a>
+          </li>
+          <li> <a href="#height" title="https://w3id.org/ontouml#height">height</a>
+          </li>
+          <li> <a href="#isAbstract" title="https://w3id.org/ontouml#isAbstract">isAbstract</a>
+          </li>
+          <li> <a href="#isComplete" title="https://w3id.org/ontouml#isComplete">isComplete</a>
+          </li>
+          <li> <a href="#isDerived" title="https://w3id.org/ontouml#isDerived">isDerived</a>
+          </li>
+          <li> <a href="#isDisjoint" title="https://w3id.org/ontouml#isDisjoint">isDisjoint</a>
+          </li>
+          <li> <a href="#isExtensional" title="https://w3id.org/ontouml#isExtensional">isExtensional</a>
+          </li>
+          <li> <a href="#isOrdered" title="https://w3id.org/ontouml#isOrdered">isOrdered</a>
+          </li>
+          <li> <a href="#isPowertype" title="https://w3id.org/ontouml#isPowertype">isPowertype</a>
+          </li>
+          <li> <a href="#isReadOnly" title="https://w3id.org/ontouml#isReadOnly">isReadOnly</a>
+          </li>
+          <li> <a href="#lowerBound" title="https://w3id.org/ontouml#lowerBound">lowerBound</a>
+          </li>
+          <li> <a href="#name" title="https://w3id.org/ontouml#name">name</a> </li>
+          <li> <a href="#order" title="https://w3id.org/ontouml#order">order</a>
+          </li>
+          <li> <a href="#text" title="https://w3id.org/ontouml#text">text</a> </li>
+          <li> <a href="#upperBound" title="https://w3id.org/ontouml#upperBound">upperBound</a>
+          </li>
+          <li> <a href="#width" title="https://w3id.org/ontouml#width">width</a>
+          </li>
+          <li> <a href="#xCoordinate" title="https://w3id.org/ontouml#xCoordinate">xCoordinate</a>
+          </li>
+          <li> <a href="#yCoordinate" title="https://w3id.org/ontouml#yCoordinate">yCoordinate</a>
+          </li>
+        </ul>
+        <h4>Annotation Properties</h4>
```

Comment:
We should remove this section. These are just the properties we use to specify our metadata. They are not part of our specification.

Reply:
Done.